src/main/resources/native/android/dalvik/source/MainActivity.java

```diff
@@ -305,6 +306,25 @@ private void notifyLifecycleEvent(String event) {
         }
     }
 
+    private static IntentHandler intentHandler;
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent intent) {
+        Log.v(TAG, "onActivityResult with requestCode " + requestCode+" and resultCode = "+resultCode+" and intent = "+intent);
+        if (intentHandler != null) {
+            intentHandler.gotActivityResult (requestCode, resultCode, intent);
+        }
+    }
+
+    public void setOnActivityResultHandler (IntentHandler handler) {
+        intentHandler = handler;
+    }
+
+
+    private static void attach_ble_startScanner() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The static method name 'attach_ble_startScanner' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5148382)